src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.CoreCLR.cs

```diff
@@ -33,6 +33,22 @@ public abstract partial class AssemblyLoadContext
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern Assembly[] GetLoadedAssemblies();
 
+        public IEnumerable<Assembly> Assemblies
```

Comment:
@marek-safar @jkotas I wasn't sure if this could be in the shared version, because it depends on `GetLoadedAssemblies();`  I do not yet fully understand the shared code concept...

Reply:
This should be in shared part

Reply:
Done